### PR TITLE
adding client status cli query

### PR DESCRIPTION
**Pull request by damiannolan**
## Description

- Adding client status cli query for existing grpc endpoint

closes: #369 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by damiannolan** [comment]
Does this require a changelog entry?

File: modules/core/02-client/client/cli/query.go [hunk 1]
```diff
@@ -89,6 +89,39 @@ func GetCmdQueryClientState() *cobra.Command {
 	return cmd
 }
 
+// GetCmdQueryClientStatus defines the command to query the status of a client with a given id
+func GetCmdQueryClientStatus() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "status [client-id]",
+		Short:   "Query client status",
+		Long:    "Query client activity status",
```
Comment (colin-axner) [inline modules/core/02-client/client/cli/query.go:97]: Maybe add `Any client without an 'Active' status is considered inactive.` to the Long?

**Review by colin-axner — APPROVED** [review]
Great work! This is a useful addition to the cli

**Comment by colin-axner** [comment]
> Does this require a changelog entry?

yup! It should probably go under `Improvements` or `Features`, whichever is a section in changelog

**Review by seantking — APPROVED** [review]

**Review by crodriguezvega — APPROVED** [review]

File: modules/core/02-client/client/cli/query.go (hunk 1 above)
Comment (damiannolan) [reply modules/core/02-client/client/cli/query.go]: Done

**Comment by damiannolan** [comment]
> yup! It should probably go under `Improvements` or `Features`, whichever is a section in changelog

Done!

**Pull request merged**